cost-analyzer/templates/monitoring-role-binding-template.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (.Values.global.platforms.openshift.enabled) }}
+{{- if (.Values.global.platforms.openshift.createMonitoringResourceReaderRoleBinding) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ template "cost-analyzer.fullname" . }}-reader
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.global.platforms.openshift.monitoringServiceAccountName | quote }}
+  namespace: openshift-monitoring
```

Comment:
Is it safe to assume that the ServiceAccount we want to bind to will always be in `openshift-monitoring`?